src/SpectatorView.Unity/Assets/SpatialAlignment/README.md

```diff
@@ -10,25 +14,77 @@ Not all spatial alignment strategies support all platforms. See the chart below
 | QR Code Detection      | x          |            | x       | x   |
 | ArUco Marker Detection |            | x          | x       | x   |
 
-## Application Flow
+## Key Concepts
+
+Before diving into the abstractions, we operate on two concepts when speaking of localization:
+
+- **Coordinate Space:** When a rotation/position is meant to be relative to a specific coordinate (location in the real world), we say it is in coordiante space. These roations/positions can be sent across the network and understood by each device that understand what coordiante they are associated with.
```

Comment:
subjective nit: I would avoid calling a coordinate real world. It may be worth calling this shared world. In VR scenarios, the same coordinate may not map 1:1 with a single physical, real world coordinate.